federated-pacman/README.md

```diff
@@ -78,29 +99,30 @@ Now that the `kubefedctl` binary has been acquired the next step is joining the
 
 Using the `kubeconfig` file that was generated, verify the Operator has been successfully deployed.
 ~~~sh
-$ oc get csv -n pacman
-NAME                DISPLAY      VERSION   REPLACES   PHASE
-federation.v0.0.10   Federation   0.0.10                Succeeded
+$ oc get csv -n kube-federation-system
+NAME                      DISPLAY            VERSION   REPLACES   PHASE
+kubefed-operator.v0.1.0   Kubefed Operator   0.1.0                Succeeded
 ~~~
+
 The next step is to federate the clusters using `kubefedctl`.
 ~~~sh
-kubefedctl join east1 --host-cluster-context east1 --add-to-registry --v=2 --federation-namespace=pacman
-kubefedctl join east2 --host-cluster-context east1 --add-to-registry --v=2 --federation-namespace=pacman
-kubefedctl join west2 --host-cluster-context east1 --add-to-registry --v=2 --federation-namespace=pacman
+kubefedctl join east1 --cluster-context east1 --host-cluster-context east2 --v=2
+kubefedctl join east2 --cluster-context east2 --host-cluster-context east2 --v=2
+kubefedctl join west2 --cluster-context west2 --host-cluster-context east2 --v=2
 
-for type in namespaces deployments.apps ingresses.extensions secrets serviceaccounts services configmaps
+for type in namespaces ingresses.extensions secrets serviceaccounts services configmaps persistentvolumeclaims deployments.apps roles.rbac.authorization.k8s.io rolebindings.rbac.authorization.k8s.io clusterrolebindings.rbac.authorization.k8s.io clusterroles.rbac.authorization.k8s.io
 do
-  kubefedctl enable $type --federation-namespace pacman
+  kubefedctl enable $type
 done
 ~~~
 
-Validate that the clusters are defined as `federatedclusters`.
+Validate that the clusters are defined as `kubefedclusters`.
```

Comment:
I believe this would be better placed above the types enablement, just after the join.